gradoop-common/src/main/java/org/gradoop/common/model/impl/properties/Property.java

```diff
@@ -167,4 +197,30 @@ public String toString() {
     return String.format("%s=%s:%s", key, value, value.getType() != null ?
       value.getType().getSimpleName() : "null");
   }
+
+  /**
+   * Returns this property as a GDL formatted String.
+   * @return A GDL formatted string that represents the property.
+   */
+  public String toGDLString() {
```

Comment:
I think the implementation should be moved to a `DataSink` or at least a separate class to keep important classes like this (and `Vertex`, `Edge`, ...) as small as possible.